Bottom: ca817f059e0163bbdb24e03f50521b1b0c742f67
Top:    a2d6a1f7cd5807bd0263c37c224c42c13c2cbbc9
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-06-06 22:03:37 +0100

Refresh of introduce-polygonhole_mode-for

---

diff --git a/src/polygon.c b/src/polygon.c
index 186f0f5..b45ae4d 100644
--- a/src/polygon.c
+++ b/src/polygon.c
@@ -275,7 +275,7 @@ ContourToPoly (PLINE * contour)
   return p;
 }
 
-POLYAREA *
+static POLYAREA *
 original_poly (PolygonType * p)
 {
   PLINE *contour = NULL;
